### ISSUE TITLE
Convert double to int in lambda task output?

### ISSUE THREAD

**Issue by arketec** [Netflix/conductor#2078]
I have a legacy server that I need to reach through a Conductor http task that only accepts numeric inputs as integers and will reject any floating point number with an error.

I searched for this issue and came across this: #1200 

But that is referring to converting an input int into a double, which is not what I am looking for. I want to convert the output into an int. 

Is this possible?

edit:
for reference, I am using the docker-compose container as a dev environment

edit2:
as a visual example:

`// input (this input comes from either a lambda task or a JSON JQ transform task)`

`{
    "id": 1
}`

`// expected serialized output (http task body)`

`"{ \"id\": 1}"`

`// actual serialized output (pulled from debugger at destination API)`

`"{\"id\": 1.0}"`

**Comment by aravindanr** [comment]
`{
  "name": "test_http_int_wf1",
  "version": 0,
  "input": {
    "id": 1
  },
  "workflowDef": {
    "name": "test_http_int_wf1",
    "description": "test_http_int_wf1",
    "version": 1,
    "tasks": [
      {
        "name": "jq_1",
        "taskReferenceName": "jq_1",
        "type": "JSON_JQ_TRANSFORM",
        "inputParameters": {
          "id": "${workflow.input.id}",
          "queryExpression": "{ id_o: .id }"
        }
      },
      {
        "name": "int_http_task",
        "taskReferenceName": "int_http_task",
        "inputParameters": {
          "http_request": {
            "uri": "https://ent0oa4isdccpe5.m.pipedream.net",
            "method": "GET",
            "contentType": "application/json",
            "accept": "application/json",
            "body": {
              "id_k": "${jq_1.output.result.id_o}"
            }
          }
        },
        "type": "HTTP"
      }
    ]
  }
}
`

I tried running the above workflow in the latest version of Conductor. You can see that the workflow input is `{"id": 1}` which is used in the body of the JSON_JQ_TRANSFORM task and the output of that task in the HTTP Task.

See screenshot below, the input is NOT converted to a Double. Am i missing some step in reproducing the issue?

![Screen Shot 2021-02-08 at 5 56 19 PM](https://user-images.githubusercontent.com/1028385/107305105-2c112a00-6a37-11eb-8e49-67c31802363e.png)

**Comment by arketec** [comment]
Thanks. Yes, I gave a simplified version of what I am doing, and when hard-coding an int, it does seem to output correctly.

So, what I did not include in my example was the step before the JQ transform which was an http response.

for example, the http response body is a simple int:

{
	"response": {
		"headers": {
			"Server": ["nginx/1.10.3 (Ubuntu)"],
			"Connection": ["keep-alive"],
			"Content-Length": ["8"],
			"Date": ["Thu, 04 Feb 2021 01:58:34 GMT"]
		},
		"reasonPhrase": "Success",
		"body": 10784123,
		"statusCode": 200
	}
}

so, replace {id: 1} with {id: "${http_step.output.response.body}"} where http_step is an http task that results in the output above
